### PR TITLE
Clean up the repository now that work has moved elsewhere

### PR DESCRIPTION
**Pull request by domenic**
This redirects people to the HTML Standard for the normative spec but preserves all the historical content and compatibility notes that were in the previous spec document.

### CODE REVIEW

**Comment by zcorpan** [comment]
I suppose this is slightly better than my https://github.com/rocallahan/innerText-spec/pull/6 :-)

**Comment by domenic** [comment]
Well, I still feel pretty silly for not checking -_-

File: README.md [hunk 1]
```diff
@@ -1,16 +1,48 @@
...
+
+The specified algorithm is depends on computed styles and on layout (due to `::first-line` affecting `text-transform`) --- `::first-line` and `::first-letter` styles are implicitly honoured. Avoiding dependence on layout would add implementation complexity (to Gecko, at least), because it's tricky to compute the correct styles for text as if `::first-line` did not apply.
+
+Special handling of `<rp>` is not in Chrome and probably not needed for Web compatibility. However the entire point of the `<rp>` element is to make plaintext rendering of ruby intelligible, so it seems churlish to not support it. As specified, an `<rp>` in a context where CSS boxes would not normally be created (e.g. inside a `<select>`) will contribute its text, but there's no simple way to avoid that. In practice this is unlikely to be a problem.
 
 ## References
+
+* [HTML Standard](https://html.spec.whatwg.org/multipage/dom.html#the-innertext-idl-attribute)
+* [Follow-up issues on the HTML Standard](https://github.com/whatwg/html/issues/1679)
+* [Web Platform Tests](https://html.spec.whatwg.org/multipage/dom.html#the-innertext-idl-attribute)
```
Comment (vsemozhetbyt) [inline README.md]: Is it OK that the first link href and the third link href are identical?

Comment (domenic) [reply README.md]: Thank you, fixed!

**Comment by domenic** [comment]
@rocallahan ping

**Comment by rocallahan** [comment]
Sorry, somehow I wasn't watching this repository...

**Comment by rocallahan** [comment]
Thanks!

**Pull request merged**